docs/metrics/dataAggregation.md

```diff
@@ -0,0 +1,8 @@
+# Data Aggregation
+
+Retina's data aggregation setting controls the metric volume produced by its agent. A lower level of aggregation yields fewer metrics, and a higher level increases them. The following table details Retina's operational behaviors at each aggregation level.
```

Comment:
Retina's data aggregation settings allows users to control the number of `Flows` or `events` generated by Retina. When set to high, Retina will serve data about the "most" important flows in a node. This allows Retina to scale to large clusters and be resource efficient. If needed, set aggregation to Low, and you will receive more detailed Flows (ex: packets at different point in the Linux kernel).